### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**

### CODE REVIEW

File: com.gitlab.newsflash.json [hunk 1]
```diff
@@ -47,8 +47,8 @@
             "sources" : [
                 {
                     "type" : "git",
-                    "url" : "https://source.puri.sm/Librem5/libhandy",
-                    "tag": "v0.0.13"
+                    "url" : "https://gitlab.gnome.org/GNOME/libhandy",
+                    "tag": "1.0.1"
```
Comment (bilelmoussaoui) [inline com.gitlab.newsflash.json:51]: No, such bump isn't possible. As the API changed between 0.0.13 & 1.0.0.

Comment (jangernert) [reply com.gitlab.newsflash.json:51]: Yup, master has been ported to libhandy1 already. But 1.0.5 doesn't support it.

Comment (TheEvilSkeleton) [reply com.gitlab.newsflash.json:51]: I see, I'll revert it

**Comment by TheEvilSkeleton** [comment]
It's working well for me.

**Pull request merged**